.ado/templates/react-native-init.yml

```diff
@@ -70,20 +70,20 @@ steps:
     inputs:
       versionSpec: ">=4.6.0"
 
+  - ${{ if eq(parameters.useNuGet, true) }}:
+    - template: prep-and-pack-nuget.yml
```

Comment:
Just curious, I don't see this downloading artifacts from universal.

Reply:
The first step of prep-and-pack-nuget.yml downloads all published artifacts under ReactWindows. A nice future optimization would be to only download what's needed, since this could also download slices unneeded for the specific job, or for desktop.